Docs/Configuration.tex

```diff
@@ -2766,126 +2775,135 @@ \section{Misc}\label{misc}
 
 \subsection{Introduction}\label{miscintro}
 
-This section contains miscellaneous configuration affecting OpenCore operating system loading behaviour
-as well as other entries, which do not go to any other section.
+This section contains miscellaneous configuration options that affect the loading behaviour of the OpenCore
+operating system in addition to other configuration options that do not readily fit into other sections.
```

Comment:
Why repeatedly say `configuration options` here? Sorry, I do not get the point.

Reply:
There is no such thing as OpenCore operating system as well. It was operating system loading behaviour.

Reply:
> Why repeatedly say configuration options here? 

"Repeatedly", which suggests multiple times, is misleading here as the phrase is only repeated once.
Overall aim has been to be as explicit as possible so that the reader never has any doubt what is being talked about at any point. A third and fourth repetition would clearly be too much but a single one helps make things clear. 

If you feel strongly about it, can change to just "options".

Reply:
> There is no such thing as OpenCore operating system as well. 

Note that was always there. Seemed odd but just left alone. Will amend.

Reply:
> Note that was always there. Seemed odd but just left alone. Will amend.

Sure, but it changed the meaning. Previously it was `[OpenCore operating system] loading behaviour` now it is `the loading behaviour of the OpenCore operating system`. I think you did not catch the fact it is rather "OpenCore's loading behaviour of the operating systems it discovers".

Reply:
`OpenCore operating system loading behaviour` is invalid gramatically which caused the confusion.
How about  `how OpenCore behaves while loading operating systems` or something to that effect?

Reply:
Restored original phrase to close out.